st2tests/st2tests/fixtures/generic/workflows/wb_invalid_syntax.yaml

```diff
@@ -1,72 +0,0 @@
-version: "2.0"
-name: "examples.mistral-complex"
```

Comment:
I don't think you need to worry about ensuring there is an Orquesta equivalent as part of this change. It would be good to have those, but there is a decent set of Orquesta examples, so don't worry too much about it right now.